From 88e9f5cd63203c3d43700294d01c22ad4945ef89 Mon Sep 17 00:00:00 2001
From: Alexander Alemayhu <alexander@alemayhu.com>
Date: Sun, 16 Oct 2016 21:09:42 +0200
Subject: [PATCH] netfilter: xtables: remove whitespace

Signed-off-by: Alexander Alemayhu <alexander@alemayhu.com>
---
 net/netfilter/xt_policy.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/netfilter/xt_policy.c b/net/netfilter/xt_policy.c
index f23e97bb42d7..25d8124b8073 100644
--- a/net/netfilter/xt_policy.c
+++ b/net/netfilter/xt_policy.c
@@ -157,7 +157,7 @@ static struct xt_match policy_mt_reg[] __read_mostly = {
 	{
 		.name		= "policy",
 		.family		= NFPROTO_IPV4,
-		.checkentry 	= policy_mt_check,
+		.checkentry	= policy_mt_check,
 		.match		= policy_mt,
 		.matchsize	= sizeof(struct xt_policy_info),
 		.me		= THIS_MODULE,
-- 
2.10.1

